Sources/XcodeGraph/DependenciesGraph/DependenciesGraph.swift

```diff
@@ -19,3 +19,178 @@ public struct DependenciesGraph: Equatable, Codable {
     /// An empty `DependenciesGraph`.
     public static let none: DependenciesGraph = .init(externalDependencies: [:], externalProjects: [:])
 }
+
+#if DEBUG
```

Comment:
What do you think of following the `Mockable` convention and hiding this behind the `MOCKING` compiler directive? That way, these symbols won't be available in non-test code that would only fail to build once you tried to build your tool in the release mode.

Reply:
Note that the `MOCKING` directive is bound to to the [Debug](https://github.com/tuist/tuist/blob/e36f83a1232a3345fe998c94ab38359b0f4afc70/Tuist/ProjectDescriptionHelpers/Module.swift#L642) configuration, so practically, the effect is the same as if I directly use the compiler directive associated to that configuration, `DEBUG`.

Reply:
I wonder if the `MOCKING` compiler directive should only be set in the test targets 🤔

Reply:
ok, let's keep things as-is then 👍